kbs/src/attestation/intel_trust_authority/mod.rs

```diff
@@ -44,8 +44,8 @@ pub enum HashAlgorithm {
 
 #[derive(Deserialize, Debug)]
 struct ItaTeeEvidence {
-    #[serde(skip)]
-    _cc_eventlog: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I believe this needs to be:
```suggestion
    #[serde(skip_deserializing_if = "Option::is_none")]
```

Reply:
Won't compile with that change - I checked with SGX&TDX using serializing_if -> works fine